bridge/slack/slack.go

```diff
@@ -285,6 +285,12 @@ func (b *Bslack) sendRTM(msg config.Message) (string, error) {
 		return "", err
 	}
 
+	// Handle prefix hint for unthreaded messages.
+	if msg.ParentID == "unthreaded" {
+		msg.ParentID = ""
+		msg.Text = fmt.Sprintf("thread reply: %s", msg.Text)
+	}
```

Comment:
I suggest using the convention/format of `[thread]` here.  It's a bit shorter, and `[prefix]` is less likely to be typed by a user as part of a regular message than `prefix: `.  It's also more consistent with existing machine-generated text like `RemoteNickFormat="[{PROTOCOL}] <{NICK}> "`.